website/docs/concepts/consensus.md

```diff
@@ -1,61 +1,50 @@
 ---
 id: consensus
-title: Consensus Mechanism
+title: Consensus
 sidebar_position: 4
 ---
 
-## What is Consensus?
+Consensus in a decentralized network refers to a state of agreement among participants on the blockchain’s history and current state. The state encompasses data such as each user's CKBytes holdings and their respective digital assets.
```

Comment:
I don't know if we should explain "consensus" here. 

We should focus on the aspects that are unique to Nervos.